app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -871,14 +871,15 @@ class AttendeeFragment : Fragment(), ComplexBackPressFragment {
     }
 
     private fun sendToken(card: Card) {
-        Stripe(requireContext())
-            .createToken(card, BuildConfig.STRIPE_API_KEY, object : TokenCallback {
-                override fun onSuccess(token: Token) {
-                    val charge = Charge(attendeeViewModel.getId().toInt(), token.id, null)
+        Stripe(requireContext(), BuildConfig.STRIPE_API_KEY)
+            .createToken(card, object : ApiResultCallback<Token> {
+                override fun onSuccess(result: Token) {
```

Comment:
Revert to token

Reply:
The interface has been changes with this new one: 
```java
public interface ApiResultCallback<ResultType> {
    void onSuccess(@NonNull ResultType result);
    void onError(@NonNull Exception e);
}
```
Normally following code conventions we should keep the implementing members in the same order as members of the interface, and keeping the methods properties names. Imagine the implementation of this method `void onError(@NonNull Exception e);` with the following variations `e`, `error`, `exception` ...

But it's not problem to change :), if you prefer

Reply:
> Normally following code conventions we should keep the implementing members in the same order as members of the interface, and keeping the methods properties names.

If that was actually true, Java wouldn't have allowed to rename or reorder the methods.

Suppose I write an interface
```kotlin
interface Result<T> {
  onSuccess(taekwondo: T)

  onError(elephant: Error)
}
```

Should the calling code use `taekwondo` and `elephant` as variable names or more appropriately named `token` and `error`

Reply:
I'm absolutely agree with you but , for me are two different things one thing is code-style and another compiler. For example `javascript` give you the option to implement a solution in different ways without warnings but that doesn't mean that all are well.

Please consult the following link: https://kotlinlang.org/docs/reference/coding-conventions.html#interface-implementation-layout  that have reference to this.